.github/workflows/linter.yml

```diff
@@ -0,0 +1,26 @@
+name: Lint Code Base
+# https://help.github.com/en/articles/workflow-syntax-for-github-actions
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  linter:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+          fetch-depth: 0 # Full history to get a proper list of changed files within `super-linter`
+
+      - name: Lint Code Base
+        uses: github/super-linter@v3
+        env:
+          VALIDATE_ALL_CODEBASE: false
+          VALIDATE_BASH:  true
+          VALIDATE_YAML:  true
+          DEFAULT_BRANCH: main
+          GITHUB_TOKEN:   ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I have opinions about key/value vertical alignment but not strong enough to work out how to configure the linter to check for it

Reply:
Yeah, YAML is rather flexible there, I think currently the linter decides. Should really add some auto-formatting somewhere ;-)